Dfe.PrepareConversions/Dfe.PrepareConversions.CypressTests/cypress/e2e/Core-journeys/application_Form_tests.cy.js

```diff
@@ -0,0 +1,95 @@
+import '../../support/commands';
+import projectTaskList from "../../pages/projectTaskList";
+const applicationFormTaskList = require('../../pages/applicationFormTaskList');
+
+describe('Test Application Form for Voluntary and Form a MAT', { tags: ['@dev', '@stage'] }, () => {
+
+    beforeEach(() => {
+        projectTaskList.getHomePage();
+    });
+
+   it('Should filter and select the project, then verify details of Voluntary', () => {
+
+    // Step 1: Filter search
+    applicationFormTaskList.filterSearch('Fahads Cypress Trust');
```

Comment:
Maybe just "Cypress Trust"?

Reply:
I need it to be Fahads Cypress Trust as Elijah has already created those cases for me. but next time I can try and change the name